### PR TITLE
Dependabot Updates for Sept 1

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

**Review by ecordell — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN apk update && apk add --no-cache git
 COPY . .
 RUN go build -v ./cmd/spicedb/
 
-FROM alpine:3.16.1
```
[the remark this answers is not in the document]
Comment (josephschorr) [reply Dockerfile:7]: I prefer to be frozen at a particular version

[the remark this answers is not in the document]
Comment (vroldanbet) [reply Dockerfile:7]: this needs to be updated to use the same base image as `Dockerfile.release`, which is already using distroless

[the remark this answers is not in the document]
Comment (vroldanbet) [reply Dockerfile:7]: opened https://github.com/authzed/spicedb/pull/797

**Pull request merged**